core/templates/dev/head/services/contextual/url.service.spec.ts

```diff
@@ -17,11 +17,12 @@
  */
 
 import { TestBed } from '@angular/core/testing';
-
 import { UrlService } from 'services/contextual/url.service';
+import { WindowRef } from './window-ref.service';
 
 describe('Url Service', () => {
-  let urlService: UrlService;
+  let urlService: UrlService = null;
+  let windowRef = null;
```

Comment:
can we add the type here as well? (Like the line above)

Reply:
Done.